homeassistant/components/epson/media_player.py

```diff
@@ -121,7 +128,7 @@ async def async_update(self):
                 self._volume = volume
         elif power_state == BUSY:
             self._state = STATE_ON
-        elif power_state == STATE_UNAVAILABLE:
+        elif power_state == EPSON_STATE_UNAVAILABLE:
             self._state = STATE_UNAVAILABLE
```

Comment:
Side note: This is incorrect. We should set the `available` entity property to `False` to mark the entity unavailable. We should not use `STATE_UNAVAILABLE` directly.

Reply:
I will prepare another PR for this to be fixed.